general/releases.rst

```diff
@@ -20,17 +20,17 @@ old, upcoming or future release.
 +---------------+--------------+
 | Version       | Release date |
 +===============+==============+
-| OP-TEE 3.15.0 | 15/Oct/21    |
+| OP-TEE 3.19.0 | 14/Oct/22    |
 +---------------+--------------+
-| OP-TEE 3.14.0 | 16/July/21   |
+| OP-TEE 3.18.0 | 15/July/22   |
 +---------------+--------------+
-| OP-TEE 3.13.0 | 16/Apr/21    |
+| OP-TEE 3.17.0 | 15/Apr/22    |
 +---------------+--------------+
-| OP-TEE 3.12.0 | 20/Jan/21    |
+| OP-TEE 3.16.0 | 14/Jan/22    |
 +---------------+--------------+
-| OP-TEE 3.11.0 | 16/Oct/20    |
+| OP-TEE 3.15.0 | 18/Oct/21    |
 +---------------+--------------+
-| OP-TEE 3.10.0 | 21/Aug/20    |
+| OP-TEE 3.14.0 | 16/July/21   |
```

Comment:
So you're not showing the dates of older releases anymore? That's fine with me, `git tag` can help with that, but then shouldn't you change the above sentence: "Starting from version 3.10.0 we track the old  and also show the releases being planned..."?

Reply:
Thanks @jforissier for noticing it. I totally missed that sentence. 
@jbech-linaro , did you have something specific in mind with this sentence - "Starting from version 3.10.0 we track the old and also show the releases being planned for the future in the table below. The dates will tell whether it is an old, upcoming or future release".
As @jforissier mentioned, I have 2 options - either i can change the sentence or add the previous releases which I deleted from the table.

Reply:
Actually when we started with this, I guess the intention was to keep old dates as well.

Reply:
I will add them back then.